### ISSUE TITLE
Missing information for the image

### ISSUE THREAD

**Issue by anna-guo-level-access** [Anna-G-s-Org/Anna-G-s-Org-Repo#19]
**Link to finding**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/manual-evaluations/69011c3564c5307d99a4e94e/results/713/view?skipScanStandards=true&status=toReview,open,fixed,partiallyFixed,notFixed,notReproducible,cannotBeFixed,validationRequested&linkedPropertyData=68faae85659492d7b1fe4d63%7C68faadce659492d7b1fe4824

**Severity**  
High

**URL**  
https://www.uab.edu/president/leadership

**Actual result**  
The content displayed in the image is not being read by the screen reader.

Important and relevant content is unavailable for screen reader users.

**Steps to reproduce**  
1. Go to the URL.
2. Navigate the page using the screen reader until reaching <the image>.

**Issue frequency**  
Individual

**Screen**  
Product

**Screenshot**  
https://download.essentialaccessibility.com/images/demo/issue%2011.png

**WCAG Success criteria**  
4.1.3

**WCAG Conformance level**  
AA

**Effort**  
Low

**Instances**  
None provided

**Disabilities Affected**  
Visual, Auditory

**Recommendation**  
Ensure the <img> has an "alt" attribute with enough and relevant information.

Best practices for alternative text:
- Less than 125 characters long.
- Don't start the alt text with expressions such as "picture of..." or "image of..." 
- Describe the image with as much detail as possible.

Refer to:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
https://www.w3schools.com/TagS/att_img_longdesc.asp
https://moz.com/learn/seo/alt-text

**Task in the Level Access Platform**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/my-tasks/6a6a1b5c6b959885e3bad8c5/6a6a26f56b959885e3bb402f

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access
